Header header
geometry_msgs/Vector3 v1
geometry_msgs/Vector3 w1
geometry_msgs/Vector3 vf
geometry_msgs/Vector3 p1
geometry_msgs/Vector3 vel1
geometry_msgs/Vector3 omega1
geometry_msgs/Vector3 p2
geometry_msgs/Vector3 pc
geometry_msgs/Vector3 realPc
geometry_msgs/Vector3 pd
geometry_msgs/Vector3 f1
geometry_msgs/Vector3 f2
geometry_msgs/Vector3 torque1
geometry_msgs/Vector3 torque2
geometry_msgs/Vector3 surfaceNormal
geometry_msgs/Vector3 surfaceTangent
float64 thetaC
float64 realThetaC
float64 thetaD
float64 dt
int32 transition
